docs/api/spaces-management/delete.asciidoc

```diff
@@ -4,7 +4,7 @@
 <titleabbrev>Delete space</titleabbrev>
 ++++
 
-Deletes a {kib} space.
+Delete a {kib} space.
```

Comment:
Delete space has a warning that it is experimental and Get features uses an experimental tag.  Let's be consistent across the API docs. I prefer the tag.

Reply:
The notes use different wording that are applicable to each API. I also prefer the tag, but don't want to get rid of important information.